readthedocs/core/urls/single_version.py

```diff
@@ -8,6 +9,7 @@
 
 from readthedocs.constants import pattern_opts
 from readthedocs.core.views import serve
+from functools import reduce
```

Comment:
import order is wrong here. should be pep8 standard of: builtins, third party, then local imports